pages/om-abakus.tsx

```diff
@@ -137,7 +137,42 @@ const OmAbakus = () => {
       </AboutCard>
       <br />
 
-      <h3 className={styles.subTitle}>Komiteer</h3>
+      <AboutCard title="Hvordan engasjere seg?">
+        <b>Komiteene og revyen</b>
+        <p>
+          Opptak til{" "}
+          <Link href="https://abakus.no/pages/grupper/104-revyen">revyen</Link>{" "}
+          og <Link href="#comittees">komiteene</Link> skjer vanligvis i
+          september etter semesterstart.{" "}
+          <Link href="https://abakus.no/pages/komiteer/5">backup</Link> har
+          vanligvis opptak i februar. \n Opptaket gjennomføres på{" "}
+          <Link href="https://opptak.abakus.no">opptak.abakus.no</Link> og vil
+          annonseres på <Link href="https://abakus.no">abakus.no</Link> og i
+          sosiale medier.
+        </p>
+        <br />
+        <b>Undergrupper</b>
+        <p>
+          Undergruppene sine opptak varierer fra gruppe til gruppe. Se hver
+          enkelt undergruppe sin side på{" "}
+          <Link href="https://abakus.no">abakus.no</Link> for mer informasjon.
```

Comment:
```suggestion
          <Link href="https://abakus.no/pages/grupper/31-undergrupper">abakus.no</Link> for mer informasjon.
```
https://abakus.no/pages/grupper/31-undergrupper has potential, but at the very least it saves people clicking the link the hassle of having to locate the info themselves (which is not that easy imo)